### PR TITLE
Setup RPC node without light client

### PR DESCRIPTION
**Pull request by scottmillner**
I've tested this and rpc node is performing as expected and rpc endpoints are functioning as expected

### CODE REVIEW

**Review by rwlockbg — APPROVED** [review]

File: bin/reth/src/commands/poa/mod.rs [hunk 1]
```diff
@@ -853,13 +841,6 @@ impl<Ext: clap::Args + fmt::Debug> PoaNodeCommand<Ext> {
             }
         };
         if !is_fed_node {
-            info!(target: "reth::cli", "Starting PoA Block Fetcher Task");
-            executor.spawn_critical(
```
Comment (arminsabouri) [inline bin/reth/src/commands/poa/mod.rs:857]: Can we also remove block fetcher as a exported member of authority-consensus?

Comment (arminsabouri) [reply bin/reth/src/commands/poa/mod.rs:857]: Then we can also remove it from the authority-builder as well

**Review by arminsabouri — APPROVED** [review]
ApproachACK

File: bin/reth/src/commands/poa/mod.rs (hunk 1 above)
Comment (scottmillner) [reply bin/reth/src/commands/poa/mod.rs:857]: done and done!

**Pull request merged**